src/main/java/AddEventCommand.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Class to present Adding event command.
+ */
+public class AddEventCommand extends Command {
+
+    private Parser parser;
+
+    /**
+     * Constructor.
+     * @param inputCommand command inputted by user.
+     * @param isExit whether to exit the chat bot or not.
+     */
+    public AddEventCommand(String inputCommand, boolean isExit, Parser parser) {
+        super(inputCommand, isExit);
+        this.parser = parser;
+    }
+
+    /**
+     * Execute the given task.
```

Comment:
In method header comments, the first sentence should start in the form verb + s (e.g. "Executes")